jobs/adspert_python_dev.html

```diff
@@ -0,0 +1,40 @@
+---
+title: Python Software Developer
+company: Adspert (Bidmanagement GmbH)
+url: https://www.adspert.net/en/jobs/python-software-developer-mf/
+location: Berlin, Germany
+contract: permanent
+contact:
+    name: Karl Bartel
+    email: jobs@adspert.net
+created: !!timestamp '2017-05-15'
+tags:
+  - python
+  - sql
+---
+We are looking for a **Python Software Developer (m/f)**
```

Comment:
Oh, of course. That's fascinating. I guess it is an analogue for words like actor/actress which have gendered equivalents in English. Many publications now avoid "actress" and just use "actor" ("female actor" where the distinction is important).